wiiu/vino/vino.ts

```diff
@@ -332,19 +419,19 @@ export default interface VinoAPI {
 	lyt_setFixedFrameSemitransparency: (unk: boolean) => void;
 
 	/**
-	 * Unknown
-	 * @param unk - Unknown
+	 * Enable or disable the white transition effect when changing page
+	 * @param enable - Enable white transition
 	 */
-	lyt_setIsEnableWhiteMask: (unk: boolean) => void;
+	lyt_setIsEnableWhiteMask: (enable: boolean) => void;
 
 	/**
-	 * Unknown
-	 * @param unk - Unknown
+	 * Enable or disable loading icon when changing page
+	 * @param enable - Enable loading icon
 	 */
-	lyt_setIsEnableClientLoadingIcon: (unk: boolean) => void;
+	lyt_setIsEnableClientLoadingIcon: (enable: boolean) => void;
 
 	/**
-	 * Unknown
+	 * Shows a touch effect on the coords of the last user touch
```

Comment:
Just a small reword suggestion

```suggestion
	 * Shows a touch effect on the position of the last user touch on the GamePad
```